Update README.md

Jugran opened the pull request.

Jugran committed:
Update README.md

Jugran committed:
Update index.html

Added Static navigation bar

chocos10 commented:
Give the screenshot or github.io link of changes made in index.html . Please elaborate in your PR what changes you want to make in the file.

Jugran commented:
I have added a fixed navigation bar which will always be on the screen, fixed.
![1](https://user-images.githubusercontent.com/32588451/32135277-f542c32c-bc19-11e7-95d9-4e71cf5b79a6.PNG)

On mouse hover, it looks something like this.
![2](https://user-images.githubusercontent.com/32588451/32135278-f573d5a2-bc19-11e7-975a-fbfa6a05e161.PNG)

Jugran committed:
Fixed Typo in navigation bar

Fixed typo mistake previously made in the navigation bar.

Jugran committed:
Navigaton Bar update

Cleaned index.html
moved navigation bar styling to style.css
